### DISCUSSION TITLE
Feature request: Enable copy for RLS ( Row Level Security)

### DISCUSSION THREAD

**Discussion by lmaloney** [apache/superset#18578] [Ideas]
I can copy Roles to make a new one and modify it, however, I can't copy an existing RLS, then modify it.

I would like to ask that for a feature, if it hasn't already been requested.

**Comment by anadue** [comment]
Agreed, this would be extremely useful. Managing multiple complex roles and RLS is a very REAL pain in superset, and it would be very helpful indeed to have a proper RLS copy function.

**Comment by LeoDiep** [comment]
+1
It would be a tedious job to do this manually again and again
